AeroBody/Body/manufacturing.md

```diff
@@ -1,4 +1,12 @@
 # Manufacturing
+
+Detailed manufacturing steps for certain parts can be reviewed here
+
+![Import Project](/_static/knowledge/Canopy_Hinge.png)
 - [Canopy Hinge](https://github.com/Solar-Gators/docs/blob/master/AeroBody/Body/PDFs/C3M_01BodyHinge.pdf)
+
+![Import Project](/_static/knowledge/Canpopy_Latch.png)
 - [Canopy Latch](https://github.com/Solar-Gators/docs/blob/master/AeroBody/Body/PDFs/C3M_03CanopyLatch.pdf)
-- [Body Hinge](https://github.com/Solar-Gators/docs/blob/master/AeroBody/Body/PDFs/C3M_01BodyHinge.pdf)
+
+![Import Project](/_static/knowledge/Body_Hinge.png)
+- [Body Hinge](https://github.com/Solar-Gators/docs/blob/master/AeroBody/Body/PDFs/C3M_01BodyHinge.pdf)
```

Comment:
Can we link this like the images are linked `[Body Hinge](/AeroBody/Body/PDFs/C3M_01BodyHinge.pdf)`